README.md

```diff
@@ -247,40 +280,36 @@ await tokenRegistry.setRoleAdmin(roleHash.RestorerRole, roleHash.RestorerAdminRo
 await tokenRegistry.setRoleAdmin(roleHash.AccepterRole, roleHash.AccepterAdminRole);
 ```
 
-Can only be called by default admin.
-
 # Deployment
 
-Hardhat is used to manage the contract development environment and deployment. This repository provides a couple of
-Hardhat tasks to simplify the deployment process.
+With Hardhat, you can manage the contract development environment and deployment. This repository provides a couple of Hardhat tasks to simplify the deployment process.
 
-Starting from v4, we have included an easy and cost-effective way to deploy the contracts while also keeping options available for advanced users to setup the contracts their preferred way.
+Starting from V4, OpenAttestation includes an easy and cost-effective way to deploy the contracts and meanwhile keep the options available for advanced users to set up the contracts in a preferable way.
```

Comment:
The V4 here is referring to the token registry (this repo) specifically, not OpenAttestation. We can change this to "Starting from v4, the token registry..."